hycom/RELO/src_2.2.98ZA-07Tsig0-i-sm-sse_relo_mpi/mod_hycom_fabm.F90

```diff
@@ -686,10 +704,42 @@ subroutine hycom_fabm_update(m, n, ibio)
       ! Vertical movement (includes sinking and floating)
       if (do_vertical_movement) then
         if (do_check_state) call check_state('before vertical_movement', n, .false.)
```

Comment:
@caglartac does not this if-stament mean that "vertical_movement() would not be called if "do_check_state" is set to false?

Reply:
.false. means it will not correct variables. The only correction done is after water column sources and sink
This is just a check if the model has infinite values

Reply:
ok, I see it is just a online statement, I thought it was a whole block.